### PR TITLE
feat(#1): Add authentication for users

### PR DESCRIPTION
**Pull request by linshi-liuting**
1. User can login to system
2. User can logout after login
3. User can register a account

Close: #1

### CODE REVIEW

**Review by linshi-wangjin — APPROVED** [review]
Look good

File: auth/logout.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User can logout now
+This is a bug in logout
```
Comment (linshi-yangkai) [inline auth/logout.txt:2]: This is a bug, fix it plz.
```suggestion
Bug is fixed
```

**Review by linshi-yangkai — COMMENTED** [review]
Plz fix the bug @linshi-liuting

File: auth/logout.txt (hunk 1 above)
Comment (linshi-liuting) [reply auth/logout.txt]: Fixed it

**Review by linshi-yangkai — APPROVED** [review]
OK

**Pull request merged**